### PR TITLE
rough outline for aspects intro

### PR DESCRIPTION
**Pull request by charles-uno**
what the main points are, grouped roughly into what the paragraphs will look like.

grabbed some ideas from the previous text, then deleted the old stuff

also iterated on "Who You Need" section

### CODE REVIEW

File: _chapters/00-outline.md [hunk 1]
```diff
@@ -21,23 +21,18 @@ The result is a punchy, quick-moving game with plenty of opportunities for memor
...
-> Gabby is the GM
+The GM and PCs are not opponents, even though the PCs and NPCs will often be enemies.
+Play is collaborative.
+Everyone at the table has the same goal: telling a good story and having a good time.
 
 ## What You Need
 
+<!-- todo: don't start on a negative. and can we do this in two paragraphs? -->
+
 {GameName} doesn't require a lot to get started:
```
Comment (agude) [inline _chapters/00-outline.md:36]:
```suggestion
{GameName} requires a few things to get started:
```

File: _chapters/00-outline.md [hunk 2]
```diff
@@ -21,23 +21,18 @@ The result is a punchy, quick-moving game with plenty of opportunities for memor
...
-> Lee/Jordan plays Little John
-> Frankie/Taylor plays Friar Tuck
-> Gabby is the GM
+The GM and PCs are not opponents, even though the PCs and NPCs will often be enemies.
+Play is collaborative.
+Everyone at the table has the same goal: telling a good story and having a good time.
 
 ## What You Need
 
+<!-- todo: don't start on a negative. and can we do this in two paragraphs? -->
```
Comment (agude) [inline _chapters/00-outline.md:34]:
```suggestion
```

File: _chapters/00-outline.md [hunk 3]
```diff
@@ -21,23 +21,18 @@ The result is a punchy, quick-moving game with plenty of opportunities for memor
...
+
 {GameName} doesn't require a lot to get started:
 just pens to write with,
```
Comment (agude) [inline _chapters/00-outline.md:37]:
```suggestion
pens to write with,
```

File: _chapters/00-outline.md [hunk 4]
```diff
@@ -76,50 +71,36 @@ TODO: Call out Evil Hat here too
 Before your first session, the group decides they want their game to be "steampunk Robin Hood".
 As it turns out, that can mean a bunch of different things. <!-- lose this line? -->
 To {A}, that means a swashbuckling adventure where bandits thumb their noses at the lecherous sheriff.
-{B} is picturing something gritty; soldiers return from a holy war to find that England no longer feels like home.
+{B} is picturing something gritty; soldiers return from a holy war to find that England no longer feels like home. <!-- is dystopian steampunk allowed? or is steampunk supposed to be optimistic? -->
```
Comment (agude) [inline _chapters/00-outline.md:74]: Sure!

Although honestly I'm not really getting into the Robin Hood grove. I'll see how today's aspect writing goes. :-/

File: _chapters/00-outline.md [hunk 5]
```diff
@@ -76,50 +71,36 @@ TODO: Call out Evil Hat here too
...
-Do we also introduce fate points here?
-Maybe mention in passing, or maybe not at all.
+At face value, an aspect is just a short phrase written down for the whole table to see.
 
-Aspects can also provide bonuses on rolls.
-More on this later.
-It all comes down to (1) what makes sense, (2) what's important to the players
+- aspects are short improvised phrases. can be about anything: people, places, things, relationships, scenes, overall setting.
+- aspects are true. otherwise, rewrite it or get rid of it.
+- aspects keep everyone on the same page about what's important. the focus of the scene. like lighting or music or director's notes in a movie. depending on the framing, FOOBAR can be tense or sad or sexy.
```
Comment (agude) [inline _chapters/00-outline.md:100]:
```suggestion
- aspects keep everyone on the same page about what's important. the focus of the scene. like lighting or music or director's notes in a movie. depending on the framing, TODO FOOBAR can be tense or sad or sexy.
```

File: _chapters/00-outline.md [hunk 6]
```diff
@@ -76,50 +71,36 @@ TODO: Call out Evil Hat here too
...
+- aspects provide scaffolding for building out the story/world. introduce things that you can come back to later: people, places, relationships, technologies. `Hunted by the Sheriff of Nottingham`, `Disheartened Veteran of the Tenth Crusade` `something flying boat something`
+- aspects provide bonuses on rolls. there's a chapter about that later. probably do NOT introduce fate points here.
+- writing good aspects. you can lean on inside jokes and pop culture.
```
Comment (agude) [inline _chapters/00-outline.md:103]: Love that we're going to have this section.

**Review by agude — APPROVED** [review]
Some commits you can integrate if you want (you can add them as a batch on the "Files changes" tab I think).

Otherwise looks good!

I mentioned in one comment I'm having some trouble with Robin Hood aspects, but we'll see what today brings.

File: _chapters/00-outline.md (hunk 4 above)
Comment (charles-uno) [reply _chapters/00-outline.md:74]: I'll avoid investing further in it for the moment. Focus on non-setting text, and maybe feel out going back to TBY and co

**Pull request merged**